src/containers/SubscriptionsPage/reducers.js

```diff
@@ -20,6 +20,6 @@ export const subscriptionsReducer = (state = fromJS({}), action = {}) => {
     case FETCH_SUBSCRIPTIONS_FAILURE:
       return state.merge({ subscriptions: [] });
     default:
-      return state;
+      return state.merge({ subscriptions: [] });
```

Comment:
I'd expect their to be an empty array as the default, but I suspect this isn't the place in the reducer to express that, since `subscription` may not be empty. How would you deal with this? Having it as a default params is probably the right way?

```javascript
export const subscriptionsReducer = (state = fromJS({subscriptions: []}), action = {}) => {
```